example/android/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "33.0.0"
-        minSdkVersion = 21
+        minSdkVersion = 22
```

Comment:
why is this updated?

Reply:
default clientId is `geddit`, which have hyperapay module (not supported on 21)